### DISCUSSION TITLE
Default values for select, radio and checkbox

### DISCUSSION THREAD

**Discussion by mastamynd** [tasteui/tasteui#712] [Q&A]
Hi All,
I’m experiencing some issues with an edit form. Specifically, I’m unable to set default values for select, radio, and checkbox inputs when not using Livewire components. Everything functions correctly when I use Livewire, but I’m struggling with standard forms.

Additionally, I’ve noticed that my select input is missing from the posted form data. To resolve this, I had to bind the select input to a hidden input field for the submission to work properly.

Could you please help me identify what I might be doing wrong?

**Comment by devajmeireles** [comment]
Are you unable to set the default selection value using the `value` property? I'd like to take a look at your Blade code. Could you please, share it here?
